.github/workflows/prepare-custom-action.yml

```diff
@@ -0,0 +1,51 @@
+name: prepare-custom-action
```

Comment:
The action should be called after what it does:  refresh-docker-image   ?

Reply:
Made it refresh-custom-action-image to keep generic than mentioning Docker directly, thank you very much!